.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-b84f79803f64354a8be3af9e5b53349f65bd7494
+22b777961074110fb8101c16a2eda7ddddbf1858
```

Comment:
Can you update the hash to cf55b254c5710cfc2aba105ef326887f8030e9f1? I have made the change. This will resolve the XLA tests failing issue. Thanks.